perfrunbook/debug_system_perf.md

```diff
@@ -125,3 +125,59 @@ When running Java applications, monitor for differences in behavior using JFR (J
   # Capture output from x86 and Graviton2 and then diff the files
   ```
 
+## Check Memory behavior via `sar -r`
+
+If your benchmark involves reading and writing to file, for example database workloads, check for "dirty pages" and "page cache" via `kbdirty`, `kbcached` respectively between two systems. This gives important information if your system is getting limited by page cache or not compared to other.
```

Comment:
We need to make this into a checklist that can be followed, and should go into the section that starts at Line 68.

Even better, add `sar -r` to the `measure_and_plot_basic_sysstat_stats.py` utility